src/pages/tutorials/build-an-nft-marketplace/index.md

```diff
@@ -431,543 +443,544 @@ This section guides you step-by-step in setting up an intuitive frontend.
-
-      const thumbnailUri = `ipfs://${responseJson.IpfsHash}`;
-      setPictureUrl(
-        `https://gateway.pinata.cloud/ipfs/${responseJson.IpfsHash}`
-      );
-
-      const op = await nftContrat!.methods
-        .mint(
-          new BigNumber(newTokenDefinition.token_id) as nat,
-          char2Bytes(newTokenDefinition.name!) as bytes,
-          char2Bytes(newTokenDefinition.description!) as bytes,
-          char2Bytes(newTokenDefinition.symbol!) as bytes,
-          char2Bytes(thumbnailUri) as bytes
-        )
-        .send();
-
-      //close directly the form
-      setFormOpen(false);
-      enqueueSnackbar(
-        "Wine collection is minting ... it will be ready on next block, wait for the confirmation message before minting another collection",
-        { variant: "info" }
-      );
-
-      await op.confirmation(2);
-
-      enqueueSnackbar("Wine collection minted", { variant: "success" });
-
-      refreshUserContextOnPageReload(); //force all app to refresh the context
-    }
-  } catch (error) {
-    console.table(`Error: ${JSON.stringify(error, null, 2)}`);
-    let tibe: TransactionInvalidBeaconError = new TransactionInvalidBeaconError(
-      error
-    );
-    enqueueSnackbar(tibe.data_message, {
-      variant: "error",
-      autoHideDuration: 10000,
-    });
-  }
-};
-```
+      } catch (error) {
+        console.table(`Error: ${JSON.stringify(error, null, 2)}`);
+        let tibe: TransactionInvalidBeaconError = new TransactionInvalidBeaconError(
+          error
+        );
+        enqueueSnackbar(tibe.data_message, {
+          variant: "error",
+          autoHideDuration: 10000,
+        });
+      }
+    };
+    ```
 
-> Note : organize/fix duplicated import declarations if necessary
+    > Note : organize/fix duplicated import declarations if necessary
 
-![mint form](/images/mintForm.png)
+    ![mint form](/images/mintForm.png)
 
-Explanations:
+    Explanations:
 
-- on Mint button click, upload a file and then call the **pinata API** to push the file to **IPFS**. It returns the hash
-- hash is used in two different ways
-  - https pinata gateway link (or any other ipfs http viewer)
-  - ipfs link for the backend thumbnail url
-- TZIP standard requires storing data in `bytes`. As there is no Michelson function to convert string to bytes (using Micheline data PACK is not working, as it alters the final bytes), do the conversion using `char2Bytes` on the frontend side
+    - on Mint button click, upload a file and then call the **pinata API** to push the file to **IPFS**. It returns the hash
+    - hash is used in two different ways
+      - https pinata gateway link (or any other ipfs http viewer)
+      - ipfs link for the backend thumbnail url
+    - TZIP standard requires storing data in `bytes`. As there is no Michelson function to convert string to bytes (using Micheline data PACK is not working, as it alters the final bytes), do the conversion using `char2Bytes` on the frontend side
```

Comment:
- No need to bold pinata or ipfs
- These bullets really need to be clarified -- there's a lot of info here that is being introduced for the first time